spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
+    configuration_script_bases = [FactoryGirl.create(:configuration_script_base),
+                                  FactoryGirl.create(:configuration_script_base)]
+    authentication1 = FactoryGirl.create(:authentication)
+    authentication1.configuration_script_bases = configuration_script_bases
+    authentication2 = FactoryGirl.create(:authentication, :configuration_script_bases => configuration_script_bases)
```

Comment:
Why create the authentication - configuration_script_base relations in different ways?